src/main/java/slimeknights/tconstruct/library/recipe/casting/container/ContainerFillingRecipe.java

```diff
@@ -99,7 +99,7 @@ public ItemStack getResultItem(RegistryAccess registryAccess) {
   @Override
   public ItemStack assemble(ICastingContainer inv, RegistryAccess registryAccess) {
     ItemStack stack = inv.getStack().copy();
-    ContainerItemContext context = ContainerItemContext.withInitial(stack);
```

Comment:
The behavior of withConstant isn't the same as withInitial so this might result in some bugs if untested